pkgs/development/libraries/tpm2-tss/default.nix

```diff
@@ -1,35 +1,28 @@
 { stdenv, lib, fetchurl, fetchpatch
-, cmocka, doxygen, ibm-sw-tpm2, iproute, openssl, perl, pkgconfig, procps
+, cmocka, doxygen, ibm-sw-tpm2, iproute, openssl, perl, pkgconfig, procps, json_c, curl
 , uthash, which
 }:
 
 stdenv.mkDerivation rec {
   pname = "tpm2-tss";
-  version = "2.3.2";
+  version = "2.4.1";
 
   src = fetchurl {
```

Comment:
It would also be nice here to use `fetchFromGitHub` to avoid manually building such a complex URL by hand.

Reply:
I'm all in favor of using `fetchFromGitHub`, unfortunately the `autoreconfHook` seems to fail for me:
```
configure.ac:21: error: possibly undefined macro: AC_SUBST
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
```

This happens for both `tss` and `abrmd`, yet to figure out a solution. In the meantime I'd vouch for using release tarballs.

Reply:
Welp, as you told me `autoconf-archive` works like a charm. Fixing it.

Reply:
Done. It was as tricky as with the `tpm2-tss`, but we got it working in the end. :tada: